rebar3/tests/test.sh

```diff
@@ -1,22 +1,18 @@
 #!/bin/sh
+#/ Usage: test.sh <pkg_ident>
+#/
+#/ Example: test.sh core/php/7.2.8/20181108151533
```

Comment:
This should probably be a version of the rebar package 😸

Reply:
I'm using the same example line in every test.sh, so its a copy and paste. Its an example of a pkg_ident, not necessarily for this specific package.